### DISCUSSION TITLE
Strategy

### DISCUSSION THREAD

**Discussion by spacechicken** [lmusarella/Pancakeswap-Prediction-Bot-2023#15] [Ideas]
Hi,
I bet $2 minimum each time and use the QUOTE_SUMMARY - lowest quote being FALSE with the martingale strategy. The maximum amount in Martingale should be $64 - you can do this by setting a a stop loss.

The Martingale strategy does not work with choosing the lowest quote - you will lose money. The STRATEGY setting is not good enough to choose a ongoing result with it - you will not make money with this setting.

Do not bet on the weekends - the volume is not there and it will skew your results.

Last week I made 500% profit - but almost lost everything as it bet $512 and I won - so you need to limit when the Martingale stops. (how many times it doubles down by putting in a stop loss amount that you are happy with, I personally wont go more than $64)

By betting on the highest you have the bet opportunity for a larger return.

**Comment by rootisareservedword** [comment]
Having the bot always choose the highest quote is not 100% reliable. While running the bot and observing PCS prediction rounds, I noticed that the bot places the bet with ~30 seconds left in the current round, so the highest quote at the time of the bet may or may not be the highest quote when the new round starts. I've also noticed that even when the round ends and begins "calculating", the higher quote for the next round can end up being the lower quote when the new round starts. 

That being said, I still am having positive results with using martingale and using the higher quote strategy, but still a lot of demo testing to be done. It seems like for this strategy to work you would always need the quote to be > 2x. I wonder if having 2 accounts using different strategies would yield positive results, or even let you achieve delta-neutral positions.

**Comment by tankiatwee** [comment]
Hey @spacechicken, do you know you can run more than 1 instance of node.js command prompt?

I dunno what took me so long but it just occur to me.

That means you can lower your base bet, run different strategies to 'hedge' and play odd AND even rounds. (Even if you edit your bot-configuration.js file, previously opened bot instances will still run based on previous config until you close the node.js window. @rootisareservedword, 1 single account will do. Alternatively, if you want to keep separate bot-configuration.js files, duplicate your PCS bot files in different sub-directories.)

The most important thing is lowering your base bet so you have more room for martingale steps while still have roughly the same time within a day to reach your profit goals.

I think the disadvantage is it pings your endpoint more frequently and since you have limited free requests, someone here recommends nodereal.io (350,000,000 monthly requests) over quicknode (10,000,000).

I'll trial this method starting tomorrow.
